### PR TITLE
Init Backend Server 

### PR DESCRIPTION
**Pull request by mike10911**
### Change list:
- Created /backend where the cabin and judging algorithms live
- init a node server in /backend
- Setup an express server in app.ts (entry point)

Note:
Still deciding if we are going to deploy with render.com or Vercel

### CODE REVIEW

File: backend/app.ts
Comment (surbhi-gulati) [inline backend/app.ts:1]: please eslint the file

File: backend/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.tnpms",
```
Comment (surbhi-gulati) [inline backend/package.json:5]: typo? assuming should be app.ts

File: backend/app.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import cors from 'cors';
+
+const app = express();
+
+app.use(express.json());
+
+app.use(
+    cors({
+        origin: 'http://localhost:3000',
```
Comment (surbhi-gulati) [inline backend/app.ts:10]: choose a different port, as this is conventionally what frontend runs on when we are operating locally

**Review by surbhi-gulati — CHANGES_REQUESTED** [review]
Cannot get the app running - can you list steps to reproduce? I just went to backend directory and put in `nodemon app.ts` but no avail :(

**Comment by mike10911** [comment]
New Changes:
- Added eslint config files
- Updated port from 3000-->4000
- Fix package.json typoes
- Add new tsconfig.json
- Added prettier config file

**Comment by mike10911** [comment]
### How to Run New Internal Tools Central Backend Server:

1. cd `backend` directory
2. run `yarn install` to install dependencies
3. run `yarn dev` to run the express server on port 4000

**Review by surbhi-gulati — APPROVED** [review]
See slack. Thanks for setting this up!!

**Review by NelsonDong41 — APPROVED** [review]
Looks good, console log is properly showing

**Pull request merged**